recipes/argoverse/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "argoverse" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/argoai/{{ name }}-api/archive/master.tar.gz
+  sha256: 8f46ba59cd28128c3a2ba35111b855e826c66e2911ea1ac779590665d6db897c
+
+build:
+  number: 0
+  skip: true                                 # [win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```